forest/bonsai/simulate_gps_data.py

```diff
@@ -89,6 +89,8 @@ def get_path(start: Tuple[float, float], end: Tuple[float, float],
     lat1, lon1 = start
     lat2, lon2 = end
     distance = great_circle_dist(lat1, lon1, lat2, lon2)
+    if not isinstance(distance, float):
+        raise TypeError("distance must be float")
```

Comment:
Are these checks actually necessary? Since we know the input type to `great_circle_dist` we can expect the correct output type.

Reply:
OK. What if `great_circle_dist` always just returns an `ndarray`?

Reply:
ok that makes sense